src/FileAdder/FileAdder.php

```diff
@@ -331,4 +324,30 @@ protected function sanitizeFileName(string $fileName) : string
     {
         return str_replace(['#', '/', '\\'], '-', $fileName);
     }
+
+    /**
+     * Attach media to the model.
```

Comment:
Let's lose the docblock: we can typehint the parameter and the function name is clear enough.